include/servant.h

```diff
@@ -79,6 +79,12 @@ extern s16 D_us_801735BC;
 // DistanceToTarget
 extern s16 D_us_801735C0;
 
+extern u32 D_us_801735C4[];
+extern u32 D_us_80170510[][3];
+extern FamiliarStats D_us_80173810;
+extern s32 D_us_80173820;
+extern Entity D_800762D8[256];
```

Comment:
PSP indicates this is `g_Entities` but it's a different memory address to that on PSX. Is that normal?

Reply:
Not sure I understand, but the memory addresses are totally different everywhere on PSP. https://github.com/Xeeynamo/sotn-decomp/blob/master/config/symbols.pspeu.txt

Reply:
`g_Entities` on PSX is `0x800733D8` but this array is `0x800762D8`.
https://github.com/Xeeynamo/sotn-decomp/blob/master/config/symbols.us.txt#L973C14-L973C24

The PSP decomp recognised this as `g_Entities` there, but it does not appear to be that same address in the PSX version, unless I'm missing something

Reply:
`0x800762D8-0x800733D8 = 0x2F00`
`0x2F00/64 = 0xBC`
sizeof(Entity) is 0xBC
https://github.com/Xeeynamo/sotn-decomp/blob/b9d7e3fd57616a495b1dbdeedd1c01e7ffd63afa/include/game.h#L857
So your address is `&g_Entities[64]`

Reply:
Aha, thank you